admin_manual/configuration/performance_tips.rst

```diff
@@ -43,3 +43,173 @@ Memcached servers must be specified in the "memcached_servers" array in ownCloud
 MySQL or MariaDB should be preferred because of the `performance limitations of SQLite with highly concurrent applications <http://www.sqlite.org/whentouse.html>`_, like ownCloud.
 
 On large instances you could consider `running MySQLTuner <https://github.com/major/MySQLTuner-perl/>`_ to optimize the database.
+
+-----------------------------------
+
+Nginx: caching owncloud gallery thumbnails with fastcgi_cache_purge
+===================================================================
+
+| One of the optimisations for owncloud when using Nginx as webserver is to combine FastCGI caching with "Cache Purge", a `3rdparty Nginx module <http://wiki.nginx.org/3rdPartyModules>`_  that adds the ability to purge content from `FastCGI`, `proxy`, `SCGI` and `uWSGI` caches. This mechanism speeds up thumbnail presentation as it shifts requests to Nginx and minimizes php invocations which else would take place for every thumbnail presented every time.
+| Following procedure is based on an Ubuntu 14.04 system. You may need to adopt it according your OS type and release.
+| **Note I:** 
+|    Unlike Apache, Nginx does not dynamically load modules. All modules needed, must be compiled into Nginx. This is one of the reasons for Nginx´s performance.
+| **Note II:**
+|    It is expected to have an already running Nginx installation with a working configuration set up like described in the owncloud documentation.
+
+Nginx module check
+------------------
+| As a first step, it is necessary to check if your Nginx installation has the ``nginx cache purge`` module compiled in.
+| 
+| ``nginx -V 2>&1 | grep ngx_cache_purge -o``
+| 
+| If your output contains ``ngx_cache_purge``, you can continue with the configuration, else you need to manually compile Nginx with the module needed.
+
+Compile Nginx with the ``nginx-cache-purge`` module
+---------------------------------------------------
+
+1. | *Preperation*
+   | ``cd /opt``
+   | ``wget http://nginx.org/keys/nginx_signing.key``
+   | ``sudo apt-key add nginx_signing.key``
+   | ``sudo vi /etc/apt/sources.list.d/nginx.list``
+   | 
+   | Add following lines (in case, replace ``{trusty}`` by your distribution name):
+   | ``deb http://nginx.org/packages/mainline/ubuntu/ trusty nginx``
+   | ``deb -src http://nginx.org/packages/mainline/ubuntu/ trusty nginx``     
+   |
+   | Then do a
+   | ``sudo apt-get update``
+   |
+   | **Note:**
+   |   If you're not overly cautious and wish to install the latest and greatest Nginx packages and features, you may have to install Nginx from it mainline repository
+   |   From the Nginx homepage: "In general, you should deploy the Nginx mainline branch at all times."
+   |
+   | If you would like to use Nginx without any additional module to be compiled in, you could do now ``sudo apt-get install nginx`` and stop:
```

Comment:
How about: If you would like to Nginx without compiling in any additional modules, just run ``sudo apt-get install nginx``.